ReverseTicTacToe/UI/ConsoleUI.cs

```diff
@@ -70,7 +252,7 @@ private  void getGameTypeFromUser()
                 userInputStr = ReadLine();
             }
             while(!isValidInput(userInputStr,1,2));
```

Comment:
why literals here?

Reply:
so maybe add them to Logic and I'll use yours, just as you added s_GameBoardMaximumDimension

Reply:
maybe instead you can use Enum.TryParse method? i think it might be more maintainable - won't have to change the constants if new game modes are added in the future